src/shell/shell.cpp

```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
 	        "REN [drive:][path]filename1 filename2.\n\n"
 	        "Note that you can not specify a new drive or path for your destination file.\n");
 	MSG_Add("SHELL_CMD_DELETE_HELP","Removes one or more files.\n");
+	MSG_Add("SHELL_CMD_DELETE_HELP_LONG", "Usage:\n"
+	        "  \033[32;1mdel\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "  \033[32;1merase\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mPATTERN\033[0m is a file name or pattern that contains wildcards (* or ?),\n"
```

Comment:
Where PATTERN can be either an exact filename or an inexact filename using one or more wildcards, which are the asterisk (*) that represents any sequence of one or more characters, and the question mark (?) that represents any single character.  For example ...

Reply:
Done.